modules/lvms-deleting-volume-clones.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-deleting-cloned-volumes_{context}"]
+= Deleting volume clones
+
+You can delete volume clones.
+[IMPORTANT]
+====
+When you delete a persistent volume claim (PVC), {lvms} deletes only the PVC but not the clones of the PVC.
```

Comment:
```suggestion
When you delete a source persistent volume claim (PVC), {lvms} deletes only the source PV but not the clones of the PVC.
```

Reply:
@radeore - Just to confirm - should it be clones of PVC instead of PV? 
In other modules related to clones & snapshots, we have mentioned that "users can clone the PVC or take a snapshot of the PVC".

Reply:
@sr1kar99 Yes we can keep, clones of "PVC" instead of "PV".